src/settings/tabtexteditor.cpp

```diff
@@ -0,0 +1,184 @@
+#include "tabtexteditor.h"
+
+#include <QHBoxLayout>
+#include <QSettings>
+#include <QTreeWidget>
+#include <QComboBox>
+#include <QLabel>
+#include <QTextEdit>
+#include <QFontDatabase>
+#include <QDebug>
```

Comment:
remove

Reply:
compiler removing #include <QDebug> in Release mode